src/datacatalogtordf/dataset.py

```diff
@@ -50,7 +50,7 @@ class Dataset(Resource):
         distributions (List[Distribution]): A list of distributions of the dataset
         frequency (URI): A link to resource describing the frequency at\
             which dataset is published.
-        spatial_coverage (Location): The geographical area covered by the dataset.
+        spatial (Location): The geographical area covered by the dataset.
```

Comment:
Tilsvarende `distributions (List[Distribution]): A list of ...`?
```suggestion
        spatial (List[Location]): The geographical area covered by the dataset.
```

Reply:
Bør det være `spatials` med `s`?

Reply:
Vi bruker samme propertynavn som i specen https://data.norge.no/specification/dcat-ap-no/#Datasett-dekningsomr%C3%A5de